Civi/Share/Api4/Action/ShareChangeMessage/ReceiveAction.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+namespace Civi\Share\Api4\Action\ShareChangeMessage;
+
+use Civi\Api4\Generic\AbstractAction;
+use Civi\Api4\Generic\Result;
+use Civi\Api4\ShareChangeMessage;
+use Civi\Share\Message;
+
+/**
+ * @method int getMessage()
+ */
+class ReceiveAction extends AbstractAction {
+
+  /**
+   * @var array
+   * A JSON-formatted change message.
+   * @required
+   */
+  protected ?array $message = NULL;
```

Comment:
Does API4 automatically unserialize JSON for this parameter?

Reply:
Yes.